core/common/src/main/java/alluxio/wire/StartupConsistencyCheckResult.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.wire;
+
+import com.google.common.base.Objects;
+
+import java.util.List;
+
+/**
+ * Result of the startup consistency check.
+ */
+public class StartupConsistencyCheckResult {
```

Comment:
After second thought, from an API user's perspective, she wants to know the consistency check's status, "result" indicates the check has finished, but it may be in progress. I'll rename it.